t/ui/15-comments.t

```diff
@@ -176,7 +176,7 @@ subtest 'URL auto-replace' => sub {
         bsc#1234 boo#2345,poo#3456 t#4567
         t#5678/modules/welcome/steps/1
         https://progress.opensuse.org/issues/6789
-        https://bugzilla.novell.com/show_bug.cgi?id=1234
+        https://bugzilla.suse.com/show_bug.cgi?id=1234
```

Comment:
ah, now I remember what issue we had. If someone copy-pastes `https://bugzilla.novell.com/show_bug.cgi?id=1234` in the comment field, previously it was replaced by `bnc#` and recorded as a bug failure. Now it's not anymore. It might be intended actually to not record this is a bugref and teach the user to not use these outdates URLs for bug referencing so I am fine with this change.